### ISSUE TITLE
first law of thermodynamics

### ISSUE THREAD

**Issue by JohnDenker** [ericmazur/PnPbook#3]
I can't seem to find the first law of thermodynamics mentioned anywhere in the text.  It would be nice to add it somewhere.  If it's already there, it should be featured more prominently.  Also it should be
added to the appropriate chapter summary, and to the index.  As it stands, in the index under T for Thermodynamics, there is an entry for the second law, but not the first.  (See #181 for a catalog of index-related issues.)

Hint:  In case anybody is wondering, I strongly suggest that the first law of thermodynamics should be defined as conservation of energy ... nothing more, nothing less.  (Details available on request.)

**Comment by ericmazur** [comment]
I guess I was SO focussed on conservation of energy, that I totally forgot about convention! See Eq. 20.1 on p.547. I guess I should add a reference there saying this is also known as the first law of thermodynamics

**Comment by ericmazur** [comment]
If I reply to this email, does it go to you, John?

I also posted a comment. Not sure if you received it.

Testing github.

Eric

**Comment by JohnDenker** [comment]
I agree with the comment posted under issue #4:  equation 20.1 ought _not_ be called the first law of thermodynamics.  (It all-too-commonly is, but it shouldn't be.)  It is only a restricted corollary.

The first law should be conservation of energy ... nothing more, nothing less.
